sumatra/recordstore/django_store/__init__.py

```diff
@@ -279,8 +279,8 @@ def clear(self):
         if not db_config.configured:
             db_config.configure()
         #management.call_command('sqlclear', 'django_store', database=self._db_label)  # this produces coloured output, need no_color option from Django 1.7
-        cmds = ["BEGIN;"] + ['DROP TABLE "django_store_{}";'.format(x)
-                             for x in ("record", "record_output_data", "record_input_data", "record_dependencies",
```

Comment:
Many thanks for the explanation. That makes a lot of sense. It sounds like a good idea in general to treat files as different if they were created in different runs (even if they have identical paths and content). Just as a precautionary question, is there any potential that this can lead to conflicts with projects upgraded from previous Sumatra versions where the table `django_store_record_output_data` was still present? I'm not familiar with how the upgrade process works in detail, so this may not be an issue.